src/Stream.re

```diff
@@ -43,15 +43,15 @@ module Writable = {
     [@bs.send] external end_: t([> kind ]) => unit = "end";
     [@bs.send] external write: (
       t([> kind ]),
-      BinaryLike.t([
+      BinaryLike.t([<
```

Comment:
Couldn't we use bs.unwrap for these cases?

Reply:
But i guess this is for consistency?

Reply:
Yeah, I was doing it for consistency. That and `@bs.unwrap` is kind of flaky sometimes. It only works for function application, which is limiting. E.g., you cannot pattern-match on the poly-variant wrappers, since they are not really supposed to exist in that case.

It's a nice syntax, so I'd be willing to try it out, but for consistency and power, I chose this instead. I'm open-minded though.

Reply:
I'm also unsure of what @bs.unwrap would do for streams across process/thread boundaries. One thing I really want is to infer the type of the data being transferred from one context to a spawned context.

E.g., if I spawn a worker thread and want to stream data into its `stdin` from the parent, I want the incoming data to be inferred. I think that's a lot easier to do with `BinaryLike.t`. But I could be overlooking something... I'd like to do some testing on this.

For now, I think we should leave it as `BinaryLike.t`, but we can keep this in the back of our minds. I do want the API to be convenient where possible.